Source/Fuse.Common/UpdateManager.uno

```diff
@@ -82,8 +81,18 @@ namespace Fuse
 		public List<UpdateListener> Onces = new List<UpdateListener>();
 		public List<UpdateListener> OncesPending = new List<UpdateListener>();
 		
-		public List<UpdateAction> PhaseDeferredActions = new List<UpdateAction>();
-		public int PhaseDeferredActionsAt = -1;
+		public Dictionary<int, Queue<UpdateAction>> PhaseDeferredActions = new Dictionary<int, Queue<UpdateAction>>();
```

Comment:
Well, technically the API allows any integer, not just the LayoutPriorities. Are you sure it is fine to make that restriction?